nondex-maven-plugin/src/it/systematic-it/verify.groovy

```diff
@@ -0,0 +1,3 @@
+File nondexDirectory = new File( basedir, ".nondex" );
+
+assert nondexDirectory.isDirectory();
```

Comment:
Add the assertion I mentioned earlier, asserting that at the end of exploration, the current equals max on all positions where the shouldExplore is true.